templates/server/index.html

```diff
@@ -1,85 +1,70 @@
+      <a href="/download/server" class="p-button--positive">Download Ubuntu Server</a>
     </div>
   </div>
-</div>
-
-<div class="p-strip is-bordered">
-  <div class="u-fixed-width">
-    <h2>What's new in Ubuntu Server {{ releases.lts.short_version }} LTS</h2>
-    <ul class="p-list is-split">
-      <li class="p-list__item is-ticked"><a href="/support">Supported by Canonical</a> for 10 years until 2032</li>
-      <li class="p-list__item is-ticked">Cloud images for <a href="/aws/pro">AWS</a>, <a href="/azure/pro">Azure</a> and <a href="/gcp/pro">GCP</a> with hardening, compliance programmes, Kernel Livepatch and more with Ubuntu Pro</li>
-      <li class="p-list__item is-ticked">Kafka, Grafana and Loki joined the growing <a href="/security/docker-images">portfolio of trusted OCI images</a>, Ubuntu-based and maintained by Canonical</li>
-      <li class="p-list__item is-ticked">OpenSSL 3.0 for modern, general-purpose cryptography and secure communication</li>
-      <li class="p-list__item is-ticked">Native host and guest drivers for NVIDIA virtual GPU (vGPU) software 14</li>
-      <li class="p-list__item is-ticked">Network acceleration improvements with SmartNIC support in Netplan</li>
-      <li class="p-list__item is-ticked">General support for GlusterFS, FRRouting and realmd/adcli under the Ubuntu main component</li>
-      <li class="p-list__item is-ticked">Runs on all major architectures: x86-64, ARM v7, ARM64, POWER9/POWER10, IBM zSystems and LinuxONE (s390x) and RISC-V</li>
-      <li class="p-list__item is-ticked">The latest long-term Linux 5.15 kernel for the recent hardware and security updates</li>
-      <li class="p-list__item is-ticked">Updates to QEMU (v6.2), libvirt (v8.0), PHP (v8.1), Ruby (v3.0), GCC (V11.2), Python (v3.10.1), MySQL (v8.0.28), OpenLDAP (v.2.5.11), Samba 4.15.5</li>
+  <div class="u-fixed-width p-section--shallow u-hide--small">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/e55cc8c0-wide-server.png",
+      alt="",
+      width="4096",
+      height="1377",
+      hi_def=True,
+      loading="auto"
+      ) | safe
+    }}
   </div>
-  <div class="u-fixed-width">
-    <p><a href="{{ releases.lts.release_notes_url }}">Read more in the release notes&nbsp;&rsaquo;</a></p>
-  </div>
-</div>
+</section>
 
-<div class="p-strip is-deep" style="padding-bottom: 2rem;">
-  <div class="u-fixed-width">
-    <h2> Performance and versatility </h2>
-    <p class="p-heading--3">Agile, secure, deploy-anywhere technology for fast-moving companies</p>
-  </div>
-  <div class="row">
-    <div class="col-6">
-      <p> It doesn't matter whether you want to deploy a NoSQL database, web farm or cloud. Certified by leading hardware OEMs and with comprehensive deployment
-        tools, so you can get the most from your infrastructure. </p>
+<section class="p-section">
+  <div class="row--50-50">
+    <hr />
```

Comment:
please add class `.p-rule` to all `<hr>`s that are not directly over a button or input field.
```suggestion
    <hr class="p-rule"/>
```